packages/contracts/contracts/L1/Colosseum.sol

```diff
@@ -417,7 +429,7 @@ contract Colosseum is Initializable, Semver {
         Types.Challenge storage challenge = challenges[_outputIndex][msg.sender];
         ChallengeStatus status = _challengeStatus(challenge);
 
-        if (_cancelIfOutputDeleted(_outputIndex, challenge.challenger, status)) {
+        if (_cancelIfOutputDeleted(_outputIndex, msg.sender, status)) {
```

Comment:
I think this should be reverted even if there are no problems with the code as of now.
With rollback to the previous one, the below `require` statement (L436-L439) can be removed.
```suggestion
        if (_cancelIfOutputDeleted(_outputIndex, challenge.challenger, status)) {
```